resources/ext/ezbake.conf

```diff
@@ -21,8 +21,9 @@ ezbake: {
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master rundir  /var/run/puppetlabs/puppetserver",
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master pidfile /var/run/puppetlabs/puppetserver/puppetserver.pid",
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master codedir /etc/puppetlabs/code",
-                 "usermod --home /opt/puppetlabs/server/data/puppetserver puppet",
-                 "install --directory --owner=puppet --group=puppet --mode=775 /opt/puppetlabs/server/data"
+                 "install --directory /etc/puppetlabs/puppet/ssl",
+                 "chown -R puppet:puppet /etc/puppetlabs/puppet/ssl",
+                 "find /etc/puppetlabs/puppet/ssl -type d -print0 | xargs chmod 770"
```

Comment:
Updated. There's not enough coffee today apparently, sorry for the churn.

It's probably not strictly necessary, more of a safety net. The case where someone has created a CA previously, this just sets everything to a known state.